packages/api-client/src/client.ts

```diff
@@ -34,12 +33,13 @@ export default class Client extends EventEmitter {
   private getHeaders = (authorization = false) => {
     const headers = {
       'Content-Type': 'application/json',
-      'X-Client-Version': this.config.getClientVersion?.(),
+      ...(this.config.extraHeaders ?? {}),
       Authorization: undefined as string | undefined,
     }
 
-    if (authorization && this.config.getAuthorization()) {
-      headers.Authorization = this.config.getAuthorization()
+    const accessToken = this.authSdk.getAccessToken()
+    if (authorization && this.authSdk.getAccessToken()) {
```

Comment:
do we need to re-invoke `getAccessToken` if you already have the variable defined above? `accessToken`